tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationUtil.java

```diff
@@ -416,6 +419,28 @@ public static boolean removeMigrationProblemsFromResource(IResource resource) {
 		return _removeProblemFromMigrationContainer(resource.getName(), container);
 	}
 
+	public static void removeProblemsInBuildForlder(MigrationProblemsContainer container) {
+		MigrationProblems[] projectProblems = container.getProblemsArray();
+
+		for (MigrationProblems migrationProblems : projectProblems) {
+			FileProblems[] fileProblems = migrationProblems.getProblems();
+
+			List<FileProblems> fileProblemsList = Stream.of(
+				fileProblems
+			).filter(
+				fileProblem -> {
+					String filePath = fileProblem.file.getPath();
+
+					return StringUtil.notContainsAll(filePath, "/build/", "/bin/", "/out/", "/target/");
```

Comment:
This should make sure that these are at <project_dir>/build/ or <project_dir>/bin/ also we should look for .project and then we know where the <project_dir> is.   You see, I'm afraid this is going to block too much.  @jtydhr88  Help @Seiphon  see how to better design this check.

Reply:
I'm going to go ahead and merge this for now.  But @Seiphon  please send another PR that makes this more correct.  Thanks.